src/ontology/mondo-edit.obo

```diff
@@ -269691,25 +269691,11 @@ property_value: seeAlso "https://rarediseases.info.nih.gov/diseases/11008/nestor
 
 [Term]
 id: MONDO:0013524
-name: bleeding diathesis due to thromboxane synthesis deficiency
-comment: This will be obsoleted in the 2023-11-01 release. The term will be split and the new ID for this term will be MONDO:0800446 (bleeding diathesis due to thromboxane synthesis deficiency (Orphanet:220443)), and MONDO:0800447 (bleeding disorder, platelet-type, 13, susceptibility to (OMIM:614009)).
-subset: obsoletion_candidate
-subset: ordo_disease {source="Orphanet:220443"}
-subset: predisposition
-synonym: "BDPLT13" RELATED ABBREVIATION [MONDO:Lexical, OMIM:614009]
-synonym: "bleeding disorder, platelet-type, 13, susceptibility to" RELATED [MONDO:Lexical, OMIM:614009]
-synonym: "bleeding disorder, susceptibility to, due to defective platelet thromboxane A2 receptor" RELATED [OMIM:614009]
-synonym: "susceptibility to platelet-type bleeding disorder 13" RELATED [OMIM:614009]
-xref: ICD10CM:D69.8 {source="Orphanet:220443/attributed", source="Orphanet:220443/ntbt", source="Orphanet:220443"}
-xref: OMIM:614009 {source="Orphanet:220443", source="MONDO:equivalentTo", source="Orphanet:220443/e"}
-xref: Orphanet:220443 {source="MONDO:equivalentTo", source="OMIM:614009"}
-xref: UMLS:C3279614 {source="OMIM:614009", source="MONDO:ncbi_mim2gene_medline", source="MONDO:notFoundInDiseaseSubset"}
-is_a: MONDO:0000009 {source="DC-OMIM:614009", source="MONDO:Redundant", source="OMIM:614009"} ! inherited bleeding disorder, platelet-type
-is_a: MONDO:0020573 {source="OMIM:614009"} ! inherited disease susceptibility
-relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/11608 {source="MONDO:mim2gene_medgen"} ! TBXA2R
-property_value: confidence "2.692307692307692" xsd:double
+name: obsolete bleeding diathesis due to thromboxane synthesis deficiency
+property_value: IAO:0000231 IAO:0000229
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/6419#issuecomment-1615253065" xsd:anyURI
-property_value: IAO:0006012 "2023-11-01" xsd:string
+is_obsolete: true
+replaced_by: MONDO:0800446
```

Comment:
This should be "consider" with both new IDs (because this obsoleted term represents 2 different terms)

Reply:
ah, right, thanks for catching that